src/Microsoft.Identity.Web.Sidecar/README.md

```diff
@@ -0,0 +1,116 @@
+# Microsoft.Identity.Web.Sidecar
+
+## Overview
+
+`Microsoft.Identity.Web.Sidecar` hosts a minimal ASP.NET Core Web API that
+enables Microsoft Entra token acquisition and downstream API calls, and token validation.
+
+### Key capabilities
+
+- Validates incoming tokens and surfaces their claims.
+- Decrypts tokens if applicable.
+- Acquires User OBO or Application tokens for configured downstream APIs.
+
+## Runtime composition
```

Comment:
I would move this section last (it's more implementation oriented than customer focused)